src/clients/api/queries/getHypotheticalPrimeApys/index.ts

```diff
@@ -0,0 +1,54 @@
+import BigNumber from 'bignumber.js';
+import { Prime } from 'packages/contracts';
+import { calculateApy } from 'utilities';
+
+import { DAYS_PER_YEAR } from 'constants/daysPerYear';
+
+export interface GetHypotheticalPrimeApysInput {
+  primeContract: Prime;
+  accountAddress: string;
+  vTokenAddress: string;
+  userBorrowBalanceMantissa: BigNumber;
+  userSupplyBalanceMantissa: BigNumber;
+  userXvsStakedMantissa: BigNumber;
+}
+
+export interface GetHypotheticalPrimeApysOutput {
+  supplyApyPercentage: BigNumber;
+  borrowApyPercentage: BigNumber;
+}
+
+const convertAprToApy = ({ aprBips }: { aprBips: string }) => {
+  // Convert bips to daily rate
+  const dailyRate = new BigNumber(aprBips).div(1000).div(DAYS_PER_YEAR);
+  // Convert daily rate to APY
+  return calculateApy({ dailyRate });
+};
+
+const getHypotheticalPrimeApys = async ({
+  primeContract,
+  vTokenAddress,
+  accountAddress,
+  userBorrowBalanceMantissa,
+  userSupplyBalanceMantissa,
+  userXvsStakedMantissa,
+}: GetHypotheticalPrimeApysInput): Promise<GetHypotheticalPrimeApysOutput> => {
+  const { borrowAPR, supplyAPR } = await primeContract.estimateAPR(
+    vTokenAddress,
+    accountAddress,
+    userBorrowBalanceMantissa.toFixed(),
+    userSupplyBalanceMantissa.toFixed(),
+    userXvsStakedMantissa.toFixed(),
+  );
+
+  // Convert APRs to daily rates
```

Comment:
nit: I think the comment is slightly off as this is calculating APYs

Reply:
Yep, completely off even hehe
Good catch